tests/suite/model/ref.typ

```diff
@@ -55,6 +55,32 @@ $ A = 1 $ <eq2>
 @arrgh
 #bibliography("/assets/bib/works.bib")
 
+--- ref-form-page ---
+#set page(numbering: "1")
+
+Text <text> is on #ref(<text>, form: "page").
+See #ref(<setup>, form: "page").
+
+#set page(supplement: [p.])
+
+== Setup <setup>
+Text seen on #ref(<text>, form: "page").
+Text seen on #ref(<text>, form: "page", supplement: "page").
+
+--- ref-form-page-unambiguous ---
+// Test that page reference is not ambiguous.
```

Comment:
what do you mean with ambiguous here?

Reply:
I wanted to mirror the `ref-ambiguous` test: you can't page reference something from the bibliography, so if a label has the same name as something from the bibliography you won't get the error in `ref-ambiguous`.

Reply:
i see